docs/operators/regexparser.md

```diff
@@ -0,0 +1,129 @@
+Regex Parser Operator
+=====================
+
+## Operator Objective
+This operator is designed to parse records based on a regex pattern and construct a concrete java class also known as ["POJO"](https://en.wikipedia.org/wiki/Plain_Old_Java_Object) out of it. User need to provide the regex pattern and schema definition to describe the data pattern. Based on regex pattern, the operator will split the data and then schema definition will be used to map the incoming record to POJO. User can also provide date format if any, in the schema. The supported constraints are listed in [constraints table](#constraints). Valid records will be emitted as POJO while invalid ones are emitted on error port with error message.
```

Comment:
User need => User needs
as POJO => as POJOs
on error => on the error
with error => with an error